app/models/ability.rb

```diff
@@ -10,6 +10,7 @@ def initialize(user)
     elsif user
       can :manage, [ResponseSet, Question, Section, Survey], status: 'published'
       can :manage, [ResponseSet, Question, Section, Survey], status: 'draft', created_by_id: user.id
+      can :manage, [ImportSession], created_by_id: user.id
```

Comment:
I think ImportSession needs to also be added to the second clause (if the user is not an admin but is a publisher then it won't hit this line so a publisher won't have ability to manage their own ImportSessions).